src/sentry/static/sentry/app/components/group/externalIssuesList.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+
+import AsyncComponent from 'app/components/asyncComponent';
+import ExternalIssueActions from 'app/components/group/externalIssueActions';
+import IssueSyncListElement from 'app/components/issueSyncListElement';
+import SentryTypes from 'app/sentryTypes';
+import PluginActions from 'app/components/group/pluginActions';
+
+class ExternalIssueList extends AsyncComponent {
+  static propTypes = {
+    group: SentryTypes.Group.isRequired,
+  };
+
+  getEndpoints() {
+    let {group} = this.props;
+    return [['integrations', `/groups/${group.id}/integrations/`]];
+  }
+
+  renderIntegrationIssues(integrations) {
+    const {group} = this.props;
+    const externalIssues = [];
+
+    if (!integrations || !integrations.length) return null;
+
+    integrations.forEach(integration => {
+      externalIssues.push(
+        <ExternalIssueActions
+          key={integration.id}
+          integration={integration}
+          group={group}
+        />
+      );
+    });
+
+    return externalIssues;
+  }
+
+  renderPluginIssues() {
+    const {group} = this.props;
+
+    return group.pluginIssues && group.pluginIssues.length
+      ? group.pluginIssues.map((plugin, i) => {
+          return <PluginActions group={group} plugin={plugin} key={i} />;
+        })
+      : null;
+  }
+
+  renderPluginActions() {
+    const {group} = this.props;
+
+    return group.pluginActions && group.pluginActions.length
```

Comment:
There isn't really a need to check the length right, since it will just map over nothing and return an empty array, which afaik is the equivalent of returning `null` as a react element.